<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKProtocolAnalyzerWidget</class>
 <widget class="QWidget" name="SAKProtocolAnalyzerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">协议分析</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: red
}</string>
     </property>
     <property name="text">
      <string>数据超过2048字节后将强制输出</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>协议起始标志</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lengthLineEdit">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="endLineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="fixedLengthCheckBox">
     <property name="text">
      <string>固定长度</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="startLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>协议结束标志</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
